.github/workflows/e2ePerformanceTests.yml

```diff
@@ -75,7 +90,7 @@ jobs:
           ARTIFACT_NAME: baseline-apk-${{ needs.prepare.outputs.VERSION }}
 
   buildDelta:
-    runs-on: ubuntu-20.04-64core
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-20.04-64core
```
Modified for testing purposes, revert change before merging to `main`.

Reply:
bump here as well